odo/backends/sql.py

```diff
@@ -309,6 +309,18 @@ def append_anything_to_sql_Table(t, c, **kwargs):
 def append_anything_to_sql_Table(t, o, **kwargs):
     return append(t, convert(Iterator, o, **kwargs), **kwargs)
 
+@append.register(sa.Table, sa.Table)
```

Comment:
should there be a test that this fails when trying to insert, say, a sqlite table into a hive table?